packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -67,8 +67,9 @@ contract InvestmentPool is IInitializableInvestmentPool, SuperAppBase, Context,
     uint256 public constant NOT_LAST_ACTIVE_MILESTONE_BYTE_VALUE = 32;
     uint256 public constant LAST_MILESTONE_BYTE_VALUE = 64;
     uint256 public constant TERMINATED_BY_VOTING_BYTE_VALUE = 128;
-    uint256 public constant SUCCESSFULLY_ENDED_BYTE_VALUE = 256;
-    uint256 public constant NO_STATE_BYTE_VALUE = 512;
+    uint256 public constant TERMINATED_BY_GELATO_BYTE_VALUE = 256;
```

Comment:
I don't think it's used in any modifiers right now, right?
Perhaps this needs a revisit?